### PR TITLE
Add tests for inheritance methods

### PR DESCRIPTION
**Pull request by jgladch**

### CODE REVIEW

**Comment by aldeed** [comment]
Will review this after merging the `inheritsHooksFrom` PR

**Comment by jgladch** [comment]
I'll have the changes ready in that PR within the hour.

File: tests.js [hunk 1]
```diff
@@ -114,3 +132,39 @@ Tinytest.add('template-extension - parentData', function (test) {
...
+  Template.testTemplate7.created();
+  test.equal(Template.testTemplate7._testTemplateField, 42);
+});
+
+Tinytest.add('template-extension - inheritsHooksFrom array', function (test) {
+  Template.testTemplate7.inheritsHooksFrom(['testTemplate', 'testTemplate8']);
+  Template.testTemplate7.created();
+  Template.testTemplate7.rendered();
+  test.equal(Template.testTemplate7._testTemplateField, 42);
+  test.equal(Template.testTemplate7._testTemplateField4, 14);
```
Comment (aldeed) [inline tests.js]: This test function should probably use something other than `testTemplate7`. The synchronous tests run in order, meaning you've already called `Template.testTemplate7.created()` in the previous test, so `_testTemplateField` is already set when this test begins. You could either clear the hooks from `testTemplate7` at the beginning of this test, or just use a new blank template.

Comment (jgladch) [inline tests.js]: Good catch, this is drudging up a couple other bugs as well. I'll fix and commit asap.

**Comment by jgladch** [comment]
Cool - this should be good to go.

**Pull request merged**